src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py

```diff
@@ -354,6 +356,88 @@ def test_azconfig_import_export(self, resource_group, location):
             exported_kvs = json.load(json_file)
         assert imported_kvs == exported_kvs
 
+        # Naming convention tests
```

Comment:
You can consider separating the test_azconfig_import_export function to several tests. In this way, you only need to re-record the corresponding feature's test.

Reply:
Done

---
In reply to: [361833424](https://github.com/Azure/azure-cli/pull/11637#discussion_r361833424) [](ancestors = 361833424)